tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1570,6 +1628,18 @@ def random_brightness(image, max_delta, seed=None):
     max_delta: float, must be non-negative.
     seed: A Python integer. Used to create a random seed. See
       `tf.compat.v1.set_random_seed` for behavior.
+    
+  Usage Example:
+    >>> x = [[[1.0, 2.0, 3.0],
+    ...       [4.0, 5.0, 6.0]],      
+    ...     [[7.0, 8.0, 9.0],      
+    ...       [10.0, 11.0, 12.0]]]
+    >>> tf.image.random_brightness(x, 0.2)
+    array([[[ 0.91842633,  1.9184263 ,  2.9184263 ],
```

Comment:
I would use only 2 significant digits for these floating points to prevent rounding errors from popping up. Instead of `0.91842633`, please write `0.91...` and so on. Similarly on the next doctest

Reply:
Fixed -> https://github.com/tensorflow/tensorflow/pull/35388/commits/76b96090d0c1d93a82c632ac8546b861e9af36ed